specs/qm/quality-management-spec.html

```diff
@@ -42,11 +42,19 @@
         {
           title: "OSLC Quality Management 2.1. Part 1: Specification (this document)",
           href: "quality-management-spec.html"
-        },
-        {
+        }
+        ,{
           title: "OSLC Quality Management 2.1. Part 2: Vocabulary",
           href: "quality-management-vocab.html"
         }
+        ,{
+          title: "OSLC Quality Management 2.1. Vocabulary RDF definitions",
+          href: "quality-management-vocab.ttl"
+        }
+        ,{
+          title: "OSLC Quality Management 2.1. Resource Shape RDF definitions",
+          href: "quality-management-shapes.ttl"
+        }
```

Comment:
@jamsden the snippet you asked me to share with you

Reply:
I'd like to get agreement on the names of th two new parts we are adding to the specs (Turtle vocabulary and shapes). @berezovskyi, you've used:
OSLC Quality Management 2.1. Part 1: Specification (this document)
OSLC Quality Management 2.1. Part 2: Vocabulary
OSLC Quality Management 2.1. Vocabulary RDF definitions
OSLC Quality Management 2.1. Resource Shape RDF definitions

In my edits for TRS, I've just put in:
OSLC TRS 3.0: Part 1: Specification
OSLC TRS 3.0: Part 2: Vocabulary
OSLC TRS 3.0: Part 3: Machine-readable RDF Vocabulary
OSLC TRS 3.0: Part 4: Machine-readable Resource Shapes

Note that Andrew does not have part numbers in the QM draft for the two new docs, but I did add part numbers for TRS. I also wanted to emphasize that the two new parts were the machine-readable sources, since we mention those in the overall OSLC Core spec.

@jamsden, @DavidJHoney - Looking for preferences and suggestions.

Reply:
@jamsden I am ready to adopt Nick's convention.

Reply:
@ndjc I just updated to adopt your convention verbatim. If anyone disagrees, they now have 2 uses to correct :)